packages/modules/cra-template-web-viewer/template.json

```diff
@@ -30,12 +30,8 @@
       "history": "^4.10.1",
       "react": "16.13.1",
       "react-dom": "16.13.1",
-      "@svgr/webpack": "4.3.3",
-      "@testing-library/jest-dom": "^4.2.4",
-      "@testing-library/react": "^10.4.4",
-      "@testing-library/user-event": "^7.1.2",
+      "@svgr/webpack": "^5.5.0",
```

Comment:
@kckst8 I don't think we need to manually depend on this ourselves. This is a direct-dep in cra, but there seems to be an issue where this pkg doesnt get installed with rush, so we need to explicitly depend on it in a rush repo, but for the everyday user this is unnecessary